src/app-layout/__integ__/app-layout-toolbar-tooltips.test.ts

```diff
@@ -0,0 +1,337 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { BasePageObject } from '@cloudscape-design/browser-test-tools/page-objects';
+import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
+
+import createWrapper from '../../../lib/components/test-utils/selectors';
+import { drawerIds as drawerIdObj } from '../../../lib/dev-pages/pages/app-layout/utils/drawer-ids';
+import { viewports } from './constants';
+
+import visualRefreshStyles from '../../../lib/components/app-layout/visual-refresh/styles.selectors.js';
+
+const wrapper = createWrapper().findAppLayout();
+class AppLayoutDrawersPage extends BasePageObject {
+  async openFirstDrawer() {
+    await this.click(wrapper.findDrawersTriggers().get(1).toSelector());
+  }
+
+  async getElementCenter(selector: string) {
+    const targetRect = await this.getBoundingBox(selector);
+    const x = Math.round(targetRect.left + targetRect.width / 2);
+    const y = Math.round(targetRect.top + targetRect.height / 2);
+    return { x, y };
+  }
+
+  async pointerDown(selector: string) {
+    const center = await this.getElementCenter(selector);
+    await (await this.browser.$(selector)).moveTo();
+    await this.browser.performActions([
+      {
+        type: 'pointer',
+        id: 'event',
+        parameters: { pointerType: 'mouse' },
+        actions: [
+          { type: 'pointerMove', duration: 0, origin: 'pointer', ...center },
+          { type: 'pointerDown', button: 0 },
+          { type: 'pause', duration: 100 },
+        ],
+      },
+    ]);
+  }
+
+  async pointerUp() {
+    await this.browser.performActions([
+      {
+        type: 'pointer',
+        id: 'event',
+        parameters: { pointerType: 'mouse' },
+        actions: [
+          { type: 'pointerUp', button: 0 },
+          { type: 'pause', duration: 100 },
+        ],
+      },
+    ]);
+  }
+}
+
+interface SetupTestOptions {
+  splitPanelPosition?: string;
+  size?: 'desktop' | 'mobile';
+  disableContentPaddings?: string;
+  visualRefresh?: string;
+}
+
+const drawerIds = Object.values(drawerIdObj);
+const VISIBLE_MOBILE_TOOLBAR_TRIGGERS_LIMIT = 2; //must match the number in  '../../../lib/components/app-layout/visual-refresh/drawers';
+const mobileDrawerTriggerIds = drawerIds.slice(0, VISIBLE_MOBILE_TOOLBAR_TRIGGERS_LIMIT);
+
+const setupTest = (
+  {
+    splitPanelPosition = 'bottom',
```

Comment:
Looks like all these params are always called with the same values except for `size`, so they are not necessary?

Reply:
removed the redundant code